administrator-manual/en/base_system2.rst

```diff
@@ -261,3 +261,49 @@ The following pages are always accessible to all users:
 * dashboard
 * applications
 * terminal
+
+Two-factor authentication (2FA)
+===============================
+
+Two-factor authentication (2FA) is an extra layer of security required to access the new Server Manager.
+First, users will enter user name and password, then they will be required to provide a temporary verification code
+generated by an application running on their smartphone.
+
+2FA is disabled by default. Each user can enable it by accessing the :guilabel:`Two-factor authentication` section
+under :guilabel:`Settings` page, then follow these steps:
+
+1. download and install the preferred 2FA application inside the smartphone
+2. scan the QR code with the 2FA application
+3. generate a new code and copy it inside :guilabel:`Verification code` field, than click :button:`Check code`
+4. if the verification code is correct, click on the :button:`Save` button
+
+Recovery codes
+--------------
+
+Recovery codes can be used instead of temporary codes if the user can't access the 2FA application on his own smartphone.
```

Comment:
Recovery codes can be used instead of temporary codes if the user can't access the 2FA application on the smartphone.